challenges/objects-arrays.js

```diff
@@ -55,11 +81,21 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for(let i = 0; i < graduates.length; i++) {
+  contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+
+for(let i = 0; i < graduates.length; i++) {
+  if(graduates[i].university ===/Uni(,)\1/) {
+  uni.push(graduates[i].university);
+  }
+}
+
 console.log(uni);
 
```

Comment:
Ok, but how many?

Reply:
Instruction didn't say to console.log how many, but I just did.